README.md

```diff
@@ -217,6 +217,7 @@ Table of Contents
   * [FraudLabs Pro](https://www.fraudlabspro.com) — Screen an order transaction for credit card payment fraud. This REST API will detect all possible fraud traits based on the input parameters of an order. Free Micro plan has 500 transactions per month.
   * [FreeGeoIP.app](https://freegeoip.app/) - Completely free Geo IP information (JSON, CSV, XML). No registration required, 15000 queries per hour rate limit.
   * [GeoDataSource](https://www.geodatasource.com) — Location search service lookup for city name by using latitude and longitude coordinate. Free API queries up to 500 times per month.
+  * [GeoIP by Lynk](https://geoip.lynk.ml) - Unlimited Requests, No Rate Throttling and Highly Accurate IP2Geo(IP Geolocation) API. Free for developers(commercial and personal). 
```

Comment:
There's no documentation or anything for this, no indication for how the api works etc - just not suitable

Reply:
+ API is on a freenom domain which is company known to redirect sites to 18+ websites

Reply:
Great, No Problem